x/dex/types/keys.go

```diff
@@ -27,7 +31,8 @@ func KeyPrefix(p string) []byte {
 }
 
 func ContractKeyPrefix(p string, contractAddr string) []byte {
-	return append([]byte(p), []byte(contractAddr)...)
+	address, _ := sdk.AccAddressFromBech32(contractAddr)
+	return append([]byte(p), address.Bytes()...)
```

Comment:
cosmos SDK uses `address.MustLengthPrefix` to pad the bytes so we probably want to do the same too https://github.com/sei-protocol/sei-cosmos/blob/main/x/bank/types/key.go#L58

Reply:
think only bank module uses that, other modules use plain accAddress bytes: https://github.com/sei-protocol/sei-cosmos/blob/main/x/auth/types/keys.go#L30-L37